docs/install-on-minikube.md

```diff
@@ -0,0 +1,70 @@
+# Install on Azure AKS
+
+## Pre-requisites
+- [Minikube](https://github.com/kubernetes/minikube)
+- To install etcd using these instructions, you must have [Helm](https://helm.sh) in your client environment.
+- [Download the Compose on Kubernetes installer](https://github.com/docker/compose-on-kubernetes/releases)
+
+## Create compose namespace
+
+- First you should have the confirmation that everything is running and `kubectl` is pointing to the minikube vm by running `minikube status`.
+If not, please start it by running `minikube start`. That should create and/or start minikube's vm.
+- Then, create a compose namespace by running `kubectl create namespace compose`
+
+## Deploy etcd
+
+If you already have an etcd instance, skip this.
+
+### Install Helm server side component
+
+- Create the tiller service account: `kubectl -n kube-system create serviceaccount tiller`
+- Give it admin access to your cluster (note: you might want to reduce the scope of this): `kubectl -n kube-system create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount kube-system:tiller`
+- Run `helm init --service-account tiller` to initialize the helm component.
+
+### Deploy etcd operator and create an etcd cluster
+
+- Run `helm install --name etcd-operator stable/etcd-operator --namespace compose` to install the etcd-operator chart
+- Create an etcd cluster definition like this one in a file named compose-etcd.yaml:
+```yaml
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdCluster"
+metadata:
+  name: "compose-etcd"
+  namespace: "compose"
+spec:
+  size: 3
+  version: "3.2.13"
+```
+- Run `kubectl apply -f compose-etcd.yaml`
+- This should bring an etcd cluster in the `compose` namespace
+
+## Deploy Compose on Kubernetes
+
+Run `installer-[darwin|linux|windows.exe] -namespace=compose -etcd-servers=http://compose-etcd-client:2379 -tag=v0.4.16 -skip-liveness-probes=true`

```

Comment:
nit: Do we need to skip the liveness probes now that https://github.com/docker/compose-on-kubernetes/pull/37 is merged?

Reply:
Cool! That works! Gonna cut it